wallabag/wallabag
# Fix double overlay on expanded right sidenav

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| Tests pass?   | yes

Before:
- Click on the filters button
- check the overlay color
- exit from this menu
- click again
- sometimes the overlay is darker

On mobile, it was even weirder because the overlay wasn't always hidden.

Why:
- JS was calling `sideNav()` for both `.button-collapse-right` buttons everytime

After:
- Only one overlay displayed
